src/MediaWiki/Page/ListBuilder.php

```diff
@@ -164,7 +164,10 @@ public function getColumnList( array $dataItems, $colsThreshold = 10 ) {
 		return $htmlColumns->getHtml();
 	}
 
-	private function buildList( $dataItems ) {
+	/**
+	 * @return array<int|string, mixed[]>
```

Comment:
This is an example of the automatically added array type. If this kind of documentation is pointless (albeit required for the CS tools) then I can rather remove these now. If somebody looks at this code later and comes up with a better type we can add it then.

Reply:
This one does not seem super helpful. I'm also concerned correctness. If Rector is taking info from existing info in docs, then it might amplify mistakes in those docs. I do not trust type docs in SMW as I've seen them be wrong multiple times in the past.